### PR TITLE
PORTALS-1667, PORTALS-1671

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

File: src/lib/utils/functions/sqlFunctions.ts [hunk 1]
```diff
@@ -78,10 +101,7 @@ export const insertConditionsFromSearchParams = (
   const tokens: string[][] = lexer.tokenize(sql)
   // we want to either create a where clause or insert into the where clause
   const foundIndex = tokens.findIndex(el => el[0] === 'WHERE')
-  const whereClauseIndex =
-    foundIndex === -1
-      ? tokens.findIndex(el => el[0] === 'FROM') + 2
```
Comment (jay-hodgson) [inline src/lib/utils/functions/sqlFunctions.ts:83]: This is one bug.  We can't assume that we should insert the WHERE token 2 tokens after the FROM token (the DOT version notation messes this up).

File: src/lib/containers/SearchV2.tsx [hunk 2]
```diff
@@ -123,11 +123,12 @@ class Search extends React.Component<InternalSearchProps, SearchState> {
     event.preventDefault()
     const { searchText } = this.state
     let { columnName } = this.state
+    const { searchable } = this.props
     if (columnName === '') {
       // default to the first one, will always be defined
       columnName =
         this.props.data?.columnModels?.filter(el =>
-          this.isSupportedColumnAndInProps(el),
```
Comment (jay-hodgson) [inline src/lib/containers/SearchV2.tsx:130]: This was a bug that I introduced.  Now isSupportedColumnAndInProps means include it if it's "searchable" (listed in the props as a search column), or (if no search columns were defined) if it supports search.  isSupportedColumn() now only tells you if it's a column type that supports search.

Comment (emmanmills) [reply src/lib/containers/SearchV2.tsx:130]: Got it.

File: src/lib/containers/SearchV2.tsx [hunk 3]
```diff
@@ -149,7 +150,7 @@ class Search extends React.Component<InternalSearchProps, SearchState> {
       const columnSingleValueQueryFilter: ColumnSingleValueQueryFilter = {
         columnName,
         operator: ColumnSingleValueFilterOperator.LIKE,
-        values: [searchText],
+        values: [`%${searchText}%`],
```
Comment (jay-hodgson) [inline src/lib/containers/SearchV2.tsx:153]: This is another bug.  The service expects the wildcard characters to get the response that we expect from the LIKE condition(s)

File: src/lib/utils/functions/sqlFunctions.ts (hunk 1 above)
Comment (emmanmills) [reply src/lib/utils/functions/sqlFunctions.ts:83]: This is interesting. Good catch!

**Review by emmanmills — APPROVED** [review]
Looks good to me. Thank you.

**Pull request merged**